src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -304,9 +302,17 @@ public EquipmentCreationEntity createSubstationEntity(String id, String name, Co
     }
 
     public VoltageLevelCreationEntity createVoltageLevelEntity(String id, String name, double nominalVoltage, String substationId,
-        List<BusbarSectionCreationEmbeddable> busbarSections,
```

Comment:
You should remove this function - we dont want functions with ALL parameters.
Callers must use DTO.toEntity() instead.